rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -513,7 +521,7 @@ struct Session {
 impl Controller {
     // TODO: Figure out how re-starting sessions automatically will work
```

Comment:
The Clients don't need to automatically restart connlib sessions, do they? This comment might be out of date.